### PR TITLE
Use MVVM Toolkit generator more

### PR DESCRIPTION
**Pull request by Sergio0694**
This PR uses `[RelayCommand]` and `[ObservableProperty]` from the MVVM Toolkit some more, where possible.

### CODE REVIEW

**Comment by dpaulino** [comment]
Is this pr going to introduce ghost bugs I have to fix in a week after deployment 🥹

**Comment by Sergio0694** [comment]
I don't think so, but only one way to find out 🤣

**Comment by dpaulino** [comment]
double checking, were you able to run ambie with this branch and it was able to play sounds without issues?

**Comment by Sergio0694** [comment]
Yup, I can play sounds just fine 🙂

**Comment by Sergio0694** [comment]
Alright I've rebased and updaated this. I've also applied two automated refactorings that were long overdue:
- Switch all files to file-scoped namespaces
- Remove unused using directives

**Comment by dpaulino** [comment]
Found an issue after just 2 seconds of testing the branch
image

can you take a look? Also, can you try to test the PR more and look for potential issues?

**Comment by Sergio0694** [comment]
Dropped the last two commits to make this easier to review, I can do those in a separate PR. Taking a look...

File: src/AmbientSounds/ViewModels/SoundViewModel.cs [hunk 1]
```diff
@@ -161,44 +156,13 @@ public bool PlusBadgeVisible
...
-            set => SetProperty(ref _downloadProgressValue, value);
-        }
-
-        public double PresenceCount
-        {
-            get => _presenceCount;
-            set
-            {
-                SetProperty(ref _presenceCount, value);
-                OnPropertyChanged(nameof(IsPresenceVisible));
```
Comment (dpaulino) [inline src/AmbientSounds/ViewModels/SoundViewModel.cs:198]: This property was designed to update visibility even when the PresenceCount value didn't change. In your PR changes, does behaviour change?

Comment (Sergio0694) [reply src/AmbientSounds/ViewModels/SoundViewModel.cs:198]: Fixed in https://github.com/jenius-apps/ambie/pull/293/commits/ae08db0206734d91404855c199193b7bfa214eb6 and made it explicit, I don't think it's good practice to notify the way it was being done earlier. Let me know what you think 🙂

**Review by dpaulino — APPROVED** [review]

**Pull request merged**